deployment/stacks/env-config.template.js

```diff
@@ -11,5 +11,4 @@ window._env_ = {
   COGNITO_REDIRECT_SIGNIN: "{{applicationEndpointUrl}}",
   COGNITO_REDIRECT_SIGNOUT: "{{applicationEndpointUrl}}",
   STAGE: "{{stage}}",
-  TIMESTAMP: "{{timestamp}}",
```

Comment:
We need to keep this. Without it, when the values don't change the file will not be recreated on deployment.

Reply:
Fixed.